apps/web/src/api/api.client.ts

```diff
@@ -1,71 +1,3 @@
-import axios from 'axios';
-import { IParamObject } from '@novu/shared';
-import { API_ROOT } from '../config';
+import { api } from '@novu/design-system';
```

Comment:
I find this move complicated to understand thinking in domain terms. I will explain myself.
I always associate a design system to the smallest pieces that help to build bigger more complicated components. And this pieces are expected to be stateless. So I am struggling to think the scenario that an HTTP client implementation would live here.
Would you mind to share the intentions with this?

Reply:
Yeah I also think it is a bit misplaced and maybe we should create client-generic package as we do have one for backends in application-generic. But I did not like to create too big of a change right now. @scopsy and @LetItRock what do you think? I could do a new refactoring on this in the cooldown.